posthog/models/organization.py

```diff
@@ -288,6 +294,13 @@ def organization_about_to_be_created(sender, instance: Organization, raw, using,
         if not is_cloud():
             instance.plugins_access_level = Organization.PluginsAccessLevel.ROOT
 
+        # Set default_anonymize_ips based on cloud deployment for new organizations
+        cloud_deployment = getattr(settings, "CLOUD_DEPLOYMENT", None)
+        if cloud_deployment == "EU":
+            instance.default_anonymize_ips = True
+        else:
+            instance.default_anonymize_ips = False
```

Comment:
nit: it's simpler this way

```suggestion
        instance.default_anonymize_ips = cloud_deployment == "EU"
```